### PR TITLE
HUB-601: Extend hub session duration

### PR DESCRIPTION
**Pull request by CharlesIC**
Extend hub session duration to reduce the proportion of people timing out. According to our research, extending the session to 2.5h from the current 90 mins will allow 20.8% of users timing out to complete their journey. We will measure this hypothesis and confirm the impact of the extension once this change has been deployed for a bit.

Details in [HUB-601](https://govukverify.atlassian.net/browse/HUB-601?focusedCommentId=25535&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-25535)

Consider alongside alphagov/verify-frontend#872

### CODE REVIEW

File: hub/config/src/main/java/uk/gov/ida/hub/config/resources/CountriesResource.java [hunk 1]
```diff
@@ -18,21 +17,16 @@
 public class CountriesResource {
 
     private final LocalConfigRepository<CountryConfig> countryConfigRepository;
-    private final ExceptionFactory exceptionFactory;
 
     @Inject
     public CountriesResource(
-            LocalConfigRepository<CountryConfig> countryConfigRepository,
-            ExceptionFactory exceptionFactory
```
Comment (galund) [inline hub/config/src/main/java/uk/gov/ida/hub/config/resources/CountriesResource.java:26]: I think this change should be a separate commit

Comment (CharlesIC) [reply hub/config/src/main/java/uk/gov/ida/hub/config/resources/CountriesResource.java:26]: `exceptionFactory` wasn't used in this class, hence removed. I really don't want to raise a separate PR for this given that it takes 2 days to get reviews

Comment (galund) [reply hub/config/src/main/java/uk/gov/ida/hub/config/resources/CountriesResource.java:26]: Agree, these things don't need separate PR's, but separate commits means the intention is more clear to anyone looking at the history, and (sometimes) we have to back changes out, which makes separating stuff cleanly quite important, I think.

Comment (CharlesIC) [reply hub/config/src/main/java/uk/gov/ida/hub/config/resources/CountriesResource.java:26]: ugh yeah separate commits is fair enough

**Comment by galund** [comment]
ta :+1:
